DEVELOPING.md

```diff
@@ -1,24 +1,33 @@
+# Developing
+
 ## Pre-requisites
 
-1.  We are using Node 8. If you need to work with multiple versions of Node, you
-    might consider using [nvm](https://github.com/creationix/nvm).
-1.  This repository uses [yarn](https://yarnpkg.com/) to manage node dependencies. Please install yarn globally using `npm install --global yarn`.
+1. We are using Node 8. If you need to work with multiple versions of Node, you
+   might consider using [nvm](https://github.com/creationix/nvm).
+2. This repository uses [yarn](https://yarnpkg.com/) to manage node
```

Comment:
Using `1.` for all the steps is typical Markdown syntax, and lets us add, remove, or reorganize steps as needed. We'd prefer to continue using this syntax rather than numbering the steps in the Markdown source.